zowex/src/comm.rs

```diff
@@ -71,12 +70,6 @@ pub fn comm_establish_connection(
     let mut cmd_to_show: String = String::new();
 
     let stream = loop {
-        let conn_result = DaemonClient::connect(daemon_socket);
-        if let Ok(good_stream) = conn_result {
-            // We made our connection. Break with the actual stream value
-            break good_stream;
-        }
```

Comment:
I think it was intentional to have this block of code first for maximum performance. If the daemon server is already running which is likely to be true for the majority of requests, then we can save time by skipping the check to determine if the process is running.

Have we benchmarked the daemon before and after to compare performance? If the difference is negligible, then I have no objection to switching to LBYL (Look Before You Leap) instead of EAFP (Easier to Ask for Forgiveness than Permission).

Reply:
Not going to benchmark, as it's clear that EAFP will be faster than LBYL, especially since it loops over system processes to find the daemon server. Addressed in upcoming commit 👍